lib/basket.rb

```diff
@@ -0,0 +1,35 @@
+class Basket
+
+  attr_reader :basket, :basket_val
+
+  def initialize
+    @basket = []
+    @basket_val = 0
+  end
+
+  def add_to_basket(item, quantity, item_price)
+    update_basket(item, quantity, item_price)
+    "#{quantity}x #{item}(s) added to your basket."
```

Comment:
This is a really useful addition to the method, gives the user confirmation and clarity in a well formatted way.